apps/src/music/progress/MusicConditions.ts

```diff
@@ -74,6 +74,11 @@ export const MusicConditions: ConditionNames = {
     description:
       'Checks if a specific sound is playing. Ex. Value: beats/drum_kit_disco',
   },
+  PLAYED_SOUND_TYPE: {
+    name: 'played_sound_type',
+    valueType: 'soundType',
```

Comment:
Agreed with Brendan that it would be ideal to avoid including `soundType` as a valueType since it's so music lab specific. One idea; we could have a value type like `array`, accompanied by an optional `values` field, so that the condition here can just supply the values directly?

Reply:
Great idea! Updated.

Reply:
Nice!